projects/1.18/assets/little-logistics/zh_cn.json

```
@@ -0,0 +1,104 @@
+{
+  "item.littlelogistics.tug": "蒸汽拖船",
+  "item.littlelogistics.barge": "货柜驳船",
+  "item.littlelogistics.chunk_loader_barge": "区块加载驳船",
+  "item.littlelogistics.seater_barge": "坐式驳船",
+  "item.littlelogistics.fishing_barge": "自动捕鱼驳船",
+  "item.littlelogistics.spring": "车链",
+  "item.littlelogistics.tug_route": "拖船航线",
+  "item.littlelogistics.locomotive_route": "机车路线",
+  "item.littlelogistics.energy_tug": "能源拖船",
+  "item.littlelogistics.fluid_barge": "液舱驳船",
+  "item.littlelogistics.fluid_car": "液体罐车",
+  "item.littlelogistics.chest_car": "货柜车厢",
+  "item.littlelogistics.chunk_loader_car": "区块加载车厢",
+  "item.littlelogistics.seater_car": "车厢",
+  "item.littlelogistics.steam_locomotive": "蒸气火车头",
+  "item.littlelogistics.energy_locomotive": "能源火车头",
+
+  "item.littlelogistics.receiver_component": "无线接收器组件",
+  "item.littlelogistics.transmitter_component": "无线发射器组件",
+  "item.littlelogistics.conductors_wrench": "导体扳手",
+
+  "entity.littlelogistics.tug": "蒸汽拖船",
+  "entity.littlelogistics.barge": "货柜驳船",
+  "entity.littlelogistics.chunk_loader_barge": "区块加载驳船",
+  "entity.littlelogistics.fishing_barge": "自动捕鱼驳船",
+  "entity.littlelogistics.spring": "车链",
+  "entity.littlelogistics.energy_tug": "能源拖船",
+  "entity.littlelogistics.fluid_barge": "液舱驳船",
+  "entity.littlelogistics.seater_barge": "坐式驳船",
+  "entity.littlelogistics.fluid_car": "液体罐车",
+  "entity.littlelogistics.chest_car": "货柜车厢",
+  "entity.littlelogistics.chunk_loader_car": "区块加载车厢",
+  "entity.littlelogistics.seater_car": "车厢",
+  "entity.littlelogistics.steam_locomotive": "蒸气火车头",
+  "entity.littlelogistics.energy_locomotive": "能源火车头",
+
+  "block.littlelogistics.tug_dock": "拖船码头",
+  "block.littlelogistics.barge_dock": "驳船码头",
+  "block.littlelogistics.guide_rail_tug": "拖船导轨",
+  "block.littlelogistics.guide_rail_corner": "船舶转角导轨",
+  "block.littlelogistics.vessel_charger": "车载充电器",
```

```suggestion
  "block.littlelogistics.vessel_charger": "交通工具充电器",
```
电力机车和电力拖船均需
此处没有好翻译（）